barretenberg/cpp/src/barretenberg/honk/proof_system/permutation_library.hpp

```diff
@@ -166,4 +166,229 @@ void compute_permutation_grand_products(std::shared_ptr<typename Flavor::Proving
     });
 }
 
+/**
+ * @brief Compute new polynomials which are the concatenated versions of other polynomials
+ *
+ * @details Multilinear PCS allow to provide openings for concatenated polynomials in an easy way by combining
```

Comment:
Could you define explicitly what you mean by "concatenation"? e.g. we have input polynomials `p1, ..., pm` and we are producing the polynomial \sum_{i=1}^m X^{n(I-1)}p_i

Reply:
Added an explanation